package.json

```diff
@@ -59,12 +59,15 @@
     ]
   },
   "dependencies": {
+    "@types/bunyan": "^1.8.0",
```

Comment:
types for flow there: https://github.com/flowtype/flow-typed/tree/master/definitions/npm/bunyan_v1.x.x